src/nix/flake.md

```diff
@@ -216,12 +216,46 @@ Currently the `type` attribute can be one of the following:
   commit hash (`rev`). Note that unlike Git, GitHub allows fetching by
   commit hash without specifying a branch or tag.
 
+  You can also specify `host` as a parameter, to point to a custom GitHub
+  Enterprise server.
```

Comment:
Please add an example. I wouldn't know how to use it given only this sentence.

Reply:
:+1:, added this to the example section just below:
```markdown
  * `github:internal/project?host=company-github.example.org`
```